sklearn/compose/_column_transformer.py

```diff
@@ -748,18 +859,30 @@ def make_column_transformer(*transformers, **kwargs):
     >>> from sklearn.compose import make_column_transformer
     >>> make_column_transformer(
     ...     (['numerical_column'], StandardScaler()),
-    ...     (['categorical_column'], OneHotEncoder()))
+    ...     (['categorical_column'], OneHotEncoder()),
+    ...     force_deprecated=False)
     ...     # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
     ColumnTransformer(n_jobs=None, remainder='drop', sparse_threshold=0.3,
              transformer_weights=None,
              transformers=[('standardscaler',
-                            StandardScaler(...),
-                            ['numerical_column']),
+                            ['numerical_column'],
+                            StandardScaler(...)),
                            ('onehotencoder',
-                            OneHotEncoder(...),
-                            ['categorical_column'])])
-
+                            ['categorical_column'],
+                            OneHotEncoder(...))])
     """
+    # Remove in v0.25, and change the default to False in v0.22
+    force_deprecated = kwargs.pop('force_deprecated', 'warn')
```

Comment:
I guess you were talking about such parameter @amueller ?

I kinda don't like that now the user needs to explicitly set this parameter to avoid a warning when using `make_column_transformer`.

Reply:
yeah I don't like it either and thing we should probably not do it?